### PR TITLE
Wms custom parameters

### PR DESCRIPTION
**Pull request by fbeqirllari**
Add custom parameters like "token" to WMS

### CODE REVIEW

**Review by jahow — CHANGES_REQUESTED** [review]
Thanks for your contribution! Custom WMS params are a great feature to have. Although the implementation is correct in my opinion, I added a couple of remarks.

Would it be possible to add tests to check whether custom params are correctly taken into account if present, and if the previous behaviour still works correctly? This can be done here: https://github.com/camptocamp/inkmap/blob/9388a2e478c35cb31bd5f39759056cca14c2f445/test/unit/printer/layers.test.js#L125

File: src/main/utils.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ export const printerReady = new Promise((resolve) => {
   } else {
     resolve(false);
   }
+  resolve(false)
```
[the remark this answers is not in the document]
Comment (jahow) [reply src/main/utils.js:45]: Sorry it looks like you resolved this comment but I still don't think this is necessary at all

**Comment by jahow** [comment]
Thanks for the adaptations! I'll fix the remaining errors and merge this if you're OK with it.

**Review by jahow — APPROVED** [review]
@fbeqirllari let me know if this can be merged, thanks!

**Review by fbeqirllari — COMMENTED** [review]
@jahow yes thanks for approving it

**Pull request merged**